Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -75,6 +82,36 @@ class SectionEditorViewController: UIViewController {
         inputViewsController.keyboardDidHide()
     }
     
+    private func setupFocusNavigationView() {
+       
+        //TODO: Localize
+        focusNavigationView.isHidden = true
+        focusNavigationView.configure(text: "Find and replace", traitCollection: traitCollection)
+        focusNavigationView.delegate = self
+        focusNavigationView.apply(theme: theme)
+        
+        focusNavigationView.translatesAutoresizingMaskIntoConstraints = false
+        navigationController?.navigationBar.addSubview(focusNavigationView)
+        
+        focusNavigationView.setNeedsLayout()
+        focusNavigationView.layoutIfNeeded()
+        
+        navigationController?.navigationBar.leadingAnchor.constraint(equalTo: focusNavigationView.leadingAnchor).isActive = true
+        navigationController?.navigationBar.trailingAnchor.constraint(equalTo: focusNavigationView.trailingAnchor).isActive = true
+        focusNavigationViewBottomConstraint = navigationController?.navigationBar.bottomAnchor.constraint(equalTo: focusNavigationView.bottomAnchor, constant: focusNavigationView.frame.height)
+        focusNavigationViewBottomConstraint.isActive = true
```

Comment:
you might be able to use  `wmf_addSubview(_, withConstraintsToEdgesWithInsets:)` here